services/catalog/service/src/common/db/in-memory-locales.ts

```diff
@@ -0,0 +1,74 @@
+import { OwnerPgPool } from '@axinom/mosaic-db-common';
+import { difference } from '@axinom/mosaic-service-common';
+import { all, deletes, insert, Queryable, select, SQL, sql } from 'zapatos/db';
+
+let inMemoryLocales: string[] = [];
+
+/**
+ * Returns a list of locales that currently exist in the system.
+ * If an API request is made for a specific locale and it is not in this array -
+ * default locale would be used instead.
+ */
+export const getInMemoryLocales = (): string[] => inMemoryLocales;
```

Comment:
Thinking of it again: this would be a problem if the Catalog Service runs in multiple instances. Maybe we can add listen/notify (https://node-postgres.com/apis/client#notification) and add triggers for insert/delete and updates (only ) in that locales table? 

And just in case if the notify fails for whatever reason we could do a background call every 5 minutes or so to update the inmemory cache?